### DISCUSSION TITLE
NodeJs学习

### DISCUSSION THREAD

**Discussion by wakaka378** [shengxinjing/it-roadmap#34] [找队友]
一个不会Node的前端不是一个好的产品经理

**Comment by wakaka378** [comment]
最近在学node框架  搞起来

**Reply by shengxinjing** [reply]
egg koa or nest?

**Reply by wakaka378** [reply]
是的  最近在学  走全栈

**Reply by jCodeLife** [reply]
最近在看大圣推荐的深入浅出node.js，快来啊

**Reply by wakaka378** [reply]
> 最近在看大圣推荐的深入浅出node.js，快来啊

这个看过一遍  很多地方都看不懂 一些深层次原理都理解不了

**Reply by wakaka378** [reply]
> 最近在看大圣推荐的深入浅出node.js，快来啊

打算先过一遍相关的框架技术  看看框架源码  然后再汇过来去看深入浅出node
